nevergrad/optimization/test_callbacks.py

```diff
@@ -112,3 +114,33 @@ def test_early_stopping() -> None:
     # below functions are inlcuded in the docstring
     assert optimizer.current_bests["minimum"].mean < 12
     assert optimizer.recommend().loss < 12  # type: ignore
+
+
+def test_optimization_logger(caplog) -> None:
+    with caplog.at_level(logging.INFO):
+        instrum = ng.p.Instrumentation(
+            None, 2.0, blublu="blublu", array=ng.p.Array(shape=(3, 2)), multiobjective=True
+        )
+        logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+        logger = logging.getLogger(__name__)
+        optimizer = optimizerlib.OnePlusOne(parametrization=instrum, budget=3)
+        optimizer.register_callback(
+            "tell",
+            callbacks.OptimizationLogger(
+                logger=logger, log_level=logging.INFO, log_interval_tells=10, log_interval_seconds=0.1
+            ),
+        )
+        optimizer.minimize(_func, verbosity=2)
+        assert (
+            "After 0, recommendation is Instrumentation(Tuple(None,2.0),Dict(array=Array{(3,2)},blublu=blublu,multiobjective=True))"
+            in caplog.text
+        )
+        assert (
+            "After 1, recommendation is Instrumentation(Tuple(None,2.0),Dict(array=Array{(3,2)},blublu=blublu,multiobjective=True))"
+            in caplog.text
+        )
+        assert (
+            "After 2, recommendation is Instrumentation(Tuple(None,2.0),Dict(array=Array{(3,2)},blublu=blublu,multiobjective=True))"
+            in caplog.text
+        )
+        assert "'multiobjective': True" in caplog.text
```

Comment:
I think one would be enough to make sure it works, no need to check in too much detail. My reason for this is that if ever we change something in the display of the parameters, I'd rather avoid needing to rewrite each of the lines :D long term maintenance is hell :D

Reply:
So just to be clear, I'll keep the line looking as is but remove the repetitions?

Reply:
yes